README.rst

```diff
@@ -184,19 +184,38 @@ Here are a snapshot of values in ``all_styles["bg-style"]`` sheet which you may
         'figure.facecolor': 'w',
         'savefig.bbox': 'tight'
 
+Get individual color and color name
+-----------------------------
+
+You may select a specific hex color code or retrieve the color name from `Colors`: ::
+
+        from bg_mpl_stylesheets.colors import Colors
+
+        # Get hex color code for bg_blue
+        bg_blue_hex = Colors.bg_blue  # Example: '#0B3C5D'
+
+        # Get color name from hex code
+        color_name = Colors.get_color_name("#0B3C5D")  # Example: 'bg_blue'
+
+        # Get a list of all bg-style colors
+        bg_colors = Colors.get_bg_colors()
+
+
 Example code
```

Comment:
I think some examples that show how it is used in matplotlib code would be good. For example, can we use the color in `pyplot.plot()?` as we would the built-in "blue"?